sn_transfers/src/cashnotes/cashnote.rs

```diff
@@ -116,16 +116,6 @@ impl CashNote {
         self.derivation_index
     }
 
-    /// Return the reason why this CashNote was spent.
-    /// Will be the default Hash (empty) if reason is none.
-    pub fn spent_reason(&self) -> Hash {
```

Comment:
Reasons now depend on `Spend` so this doesn't make sense anymore as CNs have multiple parent spends